doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,241 @@
+.. _plutus_tools_component_descriptions:
+
+Plutus tools in development 
```

Comment:
On the image, the Marconi section has a 'Haskell library' underneath it, but the description says it's an application. I think it is more accurate to think of it as a library rather than an application.

Reply:
I changed "application" to "library."